src/ConventionalCommitUtilties.js

```diff
@@ -0,0 +1,61 @@
+import ChildProcessUtilities from "./ChildProcessUtilities";
+import FileSystemUtilities from "./FileSystemUtilities";
+import logger from "./logger";
+import semver from "semver";
+
+const CHANGELOG_HEADER = `# Change Log
+
+All notable changes to this project will be documented in this file.
+See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.`;
+
+export default class ConventionalCommitUtilties {
+  @logger.logifySync()
+  static recommendVersion(name, version) {
+    try {
+      const pkgPath = `./packages/${name}/`;
+      const recommendedBump = ChildProcessUtilities.execSync(
+        `${require.resolve("conventional-recommended-bump/cli.js")} -l ${name} --commit-path=${pkgPath} -p angular`
+      );
+      return semver.inc(version, recommendedBump);
+    } catch (err) {
+      return semver.inc(version, "patch");
+    }
+  }
+
+  @logger.logifySync()
+  static updateChangelog(name, changelogLocation) {
+    try {
+      const pkgPath = `./packages/${name}/`;
+      let changelogContents = "";
+      if (FileSystemUtilities.existsSync(changelogLocation)) {
+        changelogContents = FileSystemUtilities.readFileSync(changelogLocation);
+      }
+
+      // run conventional-changelog-cli to generate the markdown
+      // for the upcoming release.
+      const newEntry = ChildProcessUtilities.execSync(
+        `${require.resolve("conventional-changelog-cli/cli.js")} -l ${name} --commit-path=${pkgPath} --pkg=${pkgPath} -p angular`
+      );
+
+      // CHANGELOG entries start with <a name=, we remove
+      // the header if it exists by starting at the first entry.
+      if (changelogContents.indexOf("<a name=") !== -1) {
```

Comment:
The header of `conventional-changelog-writer` is per release. I'm not talking about the `CHANGELOG_HEADER` here and I think this is done right. I'm just saying stripping `<a name=` manually shouldn't be needed since we could just overwrite the `headerPartial` template.